src/windowsservercore/ltsc2019/helix/amd64/Dockerfile

```diff
@@ -1,22 +1,28 @@
 # escape=`
 FROM mcr.microsoft.com/windows/servercore:ltsc2019
 
-SHELL ["cmd", "/S", "/C"]
 USER ContainerAdministrator
 
-RUN curl -SL --output %TEMP%\python.zip https://www.nuget.org/api/v2/package/python/3.7.3 `
-    && md C:\Python C:\PythonTemp `
-    && tar -zxf %TEMP%\python.zip -C C:\PythonTemp `
-    && xcopy /s c:\PythonTemp\tools C:\Python `
-    && rd /s /q c:\PythonTemp `
-    && del /q %TEMP%\python.zip `
-    && setx /M PYTHONPATH "C:\Python\Lib;C:\Python\DLLs;"
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Install latest stable version of Python
+RUN $apiUrl = 'https://api.nuget.org/v3-flatcontainer/python/index.json'; `
```

Comment:
It does feel a little strange the the SHELL is set to powershell here yet later on a direct powershell command is executed directly.  It would be nice to have consistency within this Dockerfile.